ninest/huskinfo
# Introduce `dockerfile`

This PR introduces a `dockerfile` to containerize the `yarn` app.

To test:
1. `docker build . -t test`
2. `docker run -it --name test -p 3000:3000 test`
3. `localhost:3000` should be able to show the app.
